--- origsrc/swh-plugins-0.4.15/util/blo.c	2003-03-07 17:08:21.000000000 -0600
+++ src/swh-plugins-0.4.15/util/blo.c	2008-10-10 15:17:30.861035700 -0500
@@ -54,6 +54,7 @@
 	this->table_mask = table_size - 1;
 	this->store_type = BLO_MMAP;
 
+#ifndef __CYGWIN__ /* < 1.7.0 */
 	snprintf(shm_path, 128, "/blo-1-%dx%dx%d.tbl", BLO_N_WAVES,
 			BLO_N_HARMONICS, table_size + BLO_TABLE_WR);
 	if ((shm_fd = shm_open(shm_path, O_RDONLY, 0)) > 0) {
@@ -125,6 +126,7 @@
 				MAP_SHARED, shm_fd, 0);
 		close(shm_fd);
 	}
+#endif /* CYGWIN < 1.7.0 */
 
 	/* Fallback case, can't map a SHM segment, just malloc it and suffer */
 	if (!all_tables) {
